src/components/Icon.js

```diff
@@ -2,71 +2,49 @@ import React from "react"
 import PropTypes from "prop-types"
 import styled from "styled-components"
 
+//IconContext For styling
+import { IconContext } from "react-icons"
+
+//FontAwesome React-Icon Footer Imports
+import {
+  FaGithub,
+  FaTwitterSquare,
+  FaSun,
+  FaYoutubeSquare,
+  FaSearch,
+  FaChevronDown,
+} from "react-icons/fa"
+//FontAwesome React-Icon Navbar Imports
+import {
+  FaRegArrowAltCircleRight,
+  FaExclamationCircle,
+  FaRegWindowClose,
+  FaBars,
+  FaMoon,
+  FaLanguage,
+} from "react-icons/fa"
+
 // TODO better way? FontAwesome?
 const Icon = ({ name, size, className }) => {
   return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      width={size}
-      height={size}
-      viewBox="0 0 24 24"
-      aria-labelledby={name}
-      role="presentation"
-      className={className}
+    <IconContext.Provider
+      value={{ size: size, className: className, paddingTop: "2px" }}
```

Comment:
Why are we adding `paddingTop: "2px"` here?

Reply:
I was actually struggling with the padding aspect ratio on the footer and with languages Icon when the size of Icons is small in Size. 
it should be safe to remove those now